iosApp/iosApp/Pages/Onboarding/OnboardingScreenView.swift

```diff
@@ -31,48 +50,168 @@ struct OnboardingScreenView: View {
+                            .font(Typography.title1Bold)
+                            .accessibilityHeading(.h1)
+                            .accessibilityAddTraits(.isHeader)
+                            .accessibilityFocused($focusHeader, equals: .feedback)
+                        Text(
+                            "MBTA Go is in the early stages! We want your feedback as we continue making improvements and adding new features."
+                        )
+                        .font(Typography.title3)
+                        .padding(.bottom, 16)
+                        .accessibilityHint(Text("Use the \"More\" navigation tab to send app feedback"))
+                        .dynamicTypeSize(...DynamicTypeSize.accessibility3)
+                        if typeSize >= .accessibility2, typeSize < .accessibility5 {
+                            Spacer()
+                        }
+                        Button(action: advance) {
+                            Text("Get started").onboardingKeyButton()
+                        }
+                    }
+                    .padding(.horizontal, 32)
+                    .padding(.bottom, 56)
                 }
-                Button("Show maps") {
-                    Task {
-                        try await settingUseCase.set(setting: .hideMaps, value: false)
-                        advance()
+            case .hideMaps:
+                ZStack {
+                    Image(.onboardingBackgroundMap)
+                        .resizable()
+                        .scaledToFill()
+                        .edgesIgnoringSafeArea(.all)
+                        .accessibilityHidden(true)
+                    VStack(alignment: .leading, spacing: 16) {
+                        Spacer()
+                        VStack(alignment: .leading, spacing: 16) {
+                            Text("Set map preference")
+                                .font(Typography.title1Bold)
+                                .accessibilityHeading(.h1)
+                                .accessibilityAddTraits(.isHeader)
+                                .accessibilityFocused($focusHeader, equals: .hideMaps)
+                            Text(
+                                "When using VoiceOver, we can skip reading out maps to keep you focused on transit information."
+                            )
+                            .font(Typography.title3)
+                        }
+                        .padding(32)
+                        .background(Color.fill2)
+                        .clipShape(.rect(cornerRadius: 32.0))
+                        .shadow(radius: 16)
+                        .dynamicTypeSize(...DynamicTypeSize.accessibility2)
+                        Spacer()
+                        Button(action: { Task {
+                            try await settingUseCase.set(setting: .hideMaps, value: true)
+                            advance()
+                        }}) {
```

Comment:
nitpick but it it might be nice to make a function that runs this task and takes in a boolean to toggle hideMaps. Something like:
```
func hideMaps(_ hideMaps: Bool) {
    Task { 
         try await settingUseCase.set(setting: .hideMaps, value: hideMaps)
         advance()
    }
}
 ```